### ISSUE TITLE
Add costar stabilizer support

### ISSUE THREAD

**Issue by robotmaxtron** [Ergodox-io/ergodox-case#1]
This case could easily support costar stabilizers but hasn't been implemented yet.

**Comment by yangbo** [comment]
Can the 1.5u keycaps add costar stabilizers? It's great if the improvement is implemented, I can help but don't know how to start, any ideas?

**Comment by robotmaxtron** [comment]
No, nor are they needed. This case however could easily support costar stabs for the 2u keys.
